### ISSUE TITLE
pecl install igbinary failed (php7)

### ISSUE THREAD

**Issue by kkopachev** [igbinary/igbinary#111]
> creating libtool
appending configuration tag "CXX" to libtool
configure: creating ./config.status
config.status: creating config.h
running: make
/bin/bash /tmp/pear/temp/pear-build-defaultuserUIoR4M/igbinary-2.0.0/libtool --mode=compile cc  -I. -I/tmp/pear/temp/igbinary -DPHP_ATOM_INC -I/tmp/pear/temp/pear-build-defaultuserUIoR4M/igbinary-2.0.0/include -I/tmp/pear/temp/pear-build-defaultuserUIoR4M/igbinary-2.0.0/main -I/tmp/pear/temp/igbinary -I/usr/local/include/php -I/usr/local/include/php/main -I/usr/local/include/php/TSRM -I/usr/local/include/php/Zend -I/usr/local/include/php/ext -I/usr/local/include/php/ext/date/lib  -DHAVE_CONFIG_H  -g -O2   -c /tmp/pear/temp/igbinary/src/php7/igbinary.c -o src/php7/igbinary.lo
/tmp/pear/temp/pear-build-defaultuserUIoR4M/igbinary-2.0.0/libtool: line 1283: src/php7/igbinary.loT: No such file or directory
mkdir src/php7/.libs
mkdir: cannot create directory 'src/php7/.libs': No such file or directory
Makefile:194: recipe for target 'src/php7/igbinary.lo' failed
make: *** [src/php7/igbinary.lo] Error 1
ERROR: `make' failed

Install succeeded if building from source manually.

**Comment by TysonAndre** [comment]
What was the command you used, how was pear installed, etc? I couldn't get pear installations to work.
(The DSP igbinary.dsp does not exist)

After https://github.com/igbinary/igbinary/issues/37 is fixed, and apcu builds have pecl, DLLs should be downloadable from pecl.php.net
Until then, building from source is recommended on windows

**Comment by kkopachev** [comment]
I am using official php-fmp docker image. 
Commands to repro:
```
docker run --rm -it --entrypoint=bash php:7.0-fpm 
root@435239cd8b0d:/var/www/html# pecl install igbinary
```

**Comment by kkopachev** [comment]
Thanks!

**Comment by TysonAndre** [comment]
This will be fixed whenever 2.0.1 is out. Until then, build from source. https://github.com/igbinary/igbinary/issues/110
